Nuro/Base/Lifecycle/SceneDelegate.swift

```diff
@@ -13,12 +13,19 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        if #available(iOS 14, *) {
+            if window?.traitCollection.userInterfaceIdiom == .pad {
+                if let splitViewController = createThreeColumnSplitViewController() {
+                    window?.rootViewController = splitViewController
+                }
+            }
+        }
```

Comment:
Try extracting this to another class. Minimize additional logic on `SceneDelegate`.